code/game/data_huds.dm

```diff
@@ -169,17 +169,38 @@ Medical HUD! Basic mode needs suit sensors on.
 //called when a living mob changes health
 /mob/living/proc/med_hud_set_health()
 	var/image/holder = hud_list?[HEALTH_HUD]
+	var/image/healthnumbers = hud_list?[HEALTH_HUD_NUMBERS]
 	if (isnull(holder))
 		return
-
 	holder.icon_state = "hud[RoundHealth(src)]"
 	var/icon/I = icon(icon, icon_state, dir)
 	holder.pixel_y = I.Height() - world.icon_size
+	healthnumbers.pixel_y = holder.pixel_y
+	. = healthnumbers // BUBBER EDIT ADDITION - CARRY OVER FOR SUPERCALLS
 
 //for carbon suit sensors
 /mob/living/carbon/med_hud_set_health()
-	..()
-
+// BUBBER EDIT BEGIN
+	. = ..()
+	var/image/medhud = hud_list?[HEALTH_HUD]
+
+	var/image/holder = .
+	var/string = ""
+	var/mob/living/carbon/human = src
+	var/brute = "<span style=color:red>[ROUND_UP(human.getBruteLoss())]</span>" // Limbs
+	var/burn = "<span style=color:yellow>[ROUND_UP(human.getFireLoss())]</span>" // Limbs
+	var/tox = "<span style=color:green>[ROUND_UP(human.toxloss)]</span>"
+	var/oxy = "<span style=color:cyan>[ROUND_UP(human.oxyloss)]</span>"
+	string += "<span class='center'>[brute]|[burn]|[tox]|[oxy]</span>"
+	if(medhud.icon_state == "hudhealth100")
+		holder.maptext = ""
+		return .
+	else
+		holder.maptext = MAPTEXT_SPESSFONT("<span style='font-size: 6pt; line-height: 1.0'>[string]</span>\n")
+		holder.maptext_x = -48
+		holder.maptext_y = 30
+		holder.maptext_width = 128
```

Comment:
I recommend using the RESET_TRANSFORM appearance flag to ignore all size/rotation changes from the health hud so it's readable for larger/smaller mobs.

If that doesn't work, use KEEP_APART along with it.

Reply:
``holder.appearance_flags = RESET_TRANSFORM | KEEP_APART``